packages/core/src/node/process/process.ts

```diff
@@ -19,7 +20,7 @@ export interface IProcessExitEvent {
 
 @injectable()
 export abstract class Process implements Disposable {
-
+    abstract id: number;
```

Comment:
Should not it be `readonly` and not `abstract`? How can subclasses change it?

Reply:
yes that's a mistake.. will fix